index_test.go

```diff
@@ -370,23 +369,29 @@ func TestCaseFold(t *testing.T) {
 	})
 }
 
-func TestAndSearch(t *testing.T) {
-	b := testIndexBuilder(t, nil,
+func TestSearchStats(t *testing.T) {
```

Comment:
Is there a simple way to test `noMatchTree`'s new stats? Or maybe that is handled in another test case?

Reply:
It is tested in the test case I added "one-trigram-branch-pruned"